reference/apm/cloud/apm-settings.md

```diff
@@ -47,325 +47,25 @@ If a setting is not supported by {{ech}}, you will get an error message when you
 
 ## Supported standalone APM settings (legacy) [ec-apm-settings]
```

Comment:
why doesn't ece reference/apm/cloud-enterprise/apm-settings.md have this section?